prisma/seed.ts

```diff
@@ -1,4 +1,30 @@
 import { db } from '~/modules/database/db.server';
+import { faker } from '@faker-js/faker';
+import { type User, type Password } from '@prisma/client';
+
+type UserAndPWProps = User & {
+  password: Password;
+};
+
+function createRandomUserAndPW(): UserAndPWProps {
+  const firstName = faker.person.firstName();
+  const lastName = faker.person.lastName();
+  const name = faker.internet.userName({ firstName, lastName });
+  const email = faker.internet.email({ firstName, lastName });
+  const userId = faker.string.alphanumeric(10);
```

Comment:
I would suggest aligning this with our production data and generating a random uuid here as well! Reason is that we want to keep the test data as close as possible to the production data for consistency! 

If we ever have a prod vs. test database, then we won't need to distinguish test users from prod users anyways. And IF we still want to distinguish, then a user.type property would probably be better!